lib/tilt/livescript.rb

```diff
@@ -13,7 +13,7 @@ def prepare
     end
 
     def evaluate(scope, locals, &block)
-      @output ||= LiveScript.compile(data, options)
+      @output ||= LiveScript.compile(data, {bare: true, header: false})
```

Comment:
This should at least be `{bare: true, header: false}.merge(options)` so you can override the options, but in general I'm not a fan of adding Tilt-defaults. The LiveScript library already has its own defaults.

Reply:
yeah i was thinking the same tbh